atc/db/resource_cache_lifecycle.go

```diff
@@ -130,25 +130,3 @@ func (f *resourceCacheLifecycle) CleanUpInvalidCaches(logger lager.Logger) error
 
 	return nil
 }
-
-func (f *resourceCacheLifecycle) CleanUsesForPausedPipelineResources() error {
```

Comment:
I was like what, why delete?
`ag CleanUsesForPausedPipelineResources`
0 hits
ah unused, nice.

For some reason staticcheck didn't flag this func 🤔